pkg/build/build.go

```diff
@@ -0,0 +1,10 @@
+package build
+
+// commitFromGit is a constant representing the source version that
+// generated this build. It should be set during build via -ldflags.
+var commitFromGit string
```

Comment:
Doesn't `-X` only work with exported variables? :thinking:

Reply:
Okay. I think that the "it seems to work" conversation could be easily settled by a test, though 😅.

A test would also help find regressions in the future. My experience with such brittle settings is that any refactors (either on the build system side, or moving packages around) easily lead to silent failures.

Suggestion: if it's feasible, implement a regression test verifying that the variable contains something reasonable.

Reply:
Good catch, let's add a job for that then: https://github.com/scylladb/scylla-operator-release/pull/509